src/types.ts

```diff
@@ -43,6 +43,7 @@ export interface IProps<Value> {
   removableByMove: boolean;
   lockVertically: boolean;
   voiceover: IVoiceover;
+  container?: Element;
```

Comment:
```suggestion
  container?: Element | null;
```

so it works my proposed change above.